ui/src/app/components/webcomponents/test-plan-platform-os-version-form.component.ts

```diff
@@ -154,29 +154,31 @@ export class TestPlanPlatformOsVersionFormComponent implements OnInit {
   fetchBrowsers(setValue?: Boolean) {
     this.platformService.findAllBrowsers(this.platform, this.platformOsVersion, this.version.workspace.workspaceType, this.testPlanLabType).subscribe(res => {
       this.browsers = res;
-      if (!this.isPrivateGrid) {
-        if (this.environmentFormGroup?.controls['browser']?.value){
-          this.platformBrowser = this.browsers.find(browser => browser.id == this.environmentFormGroup?.controls['browser']?.value);
-        }
-        if (!this.platformBrowser || setValue) {
-          this.platformBrowser = this.browsers[0];
-          this.environmentFormGroup?.controls['browser']?.setValue(this.platformBrowser?.id);
-          if (this.version.workspace.isMobileWeb && this.platformBrowser.isChrome) {
-            this.environmentFormGroup?.controls['browser']?.setValue(WebBrowser.CHROME);
+      if(!this.isHybrid){
```

Comment:
@Raksha-testsigma Can you explain why we are not fetching browsers if it is hybrid execution ?

Reply:
While we open the Adhoc Run window, it by default opens with the selection of Testsigma Labs, which sets the browser and version for the Lab. During this, the Local Device browser name in the Local executions disappears due to a network delay. Hence we add a check to ensure that it does not reset but displays the browser name constantly.